samples: nrf9160: gnss: rename sample and add nRF Cloud A-GPS/P-GPS support (#6010)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Added a new shell command ``rest`` for sending simple REST requests and receiving responses to them.

* :ref:`gnss_sample` sample:

>   * Renamed from GPS sample.

Reviewer: ```suggestion
  * Renamed. The previous name was nRF9160: GPS with SUPL client library.
```

  * Added support for nRF Cloud A-GPS and P-GPS.

>   * LTE now remains connected to the network all the time when assistance is enabled. It's possible to enable old behavior with A-GPS using a Kconfig option.

Reviewer: ```suggestion
  * LTE now remains connected to the network all the time when assistance is enabled. You can enable the old behavior with A-GPS using a Kconfig option.
```
